ds-and-algo/answers/q4.html

```diff
@@ -0,0 +1,68 @@
+<html>
+  <head>
+    <title>Data Structures And Algorithms | InterQues</title>
+	
+	<meta
+      name="description"
+      content="How to Reverse a Linked List ?"
+    />
+  </head>
+  <body>
+    <div>
+      <img
+        src="../../assets/hacktoberfest.png"
+        height="120px"
+        alt="Hacktoberfest"
+      />
+    </div>
+
+    <div>
+      <p>
+        <small>
+          <a href="../../index.html">Home</a>
+          &gt;
+          <a href="../index.html"> Data Structures and Algorithms Questions</a>
+          &gt; How to Reverse a Linked List ?
+          <!-- Your Question Title -->
```

Comment:
Insert your question title here too.

Reply:
It's already their bro, I just forgot to remove the comment, the question is there in the 25th line of code. I hope I cleared your doubt

Reply:
Oops, my bad.
Please remove the comment then.